src/com/googlecode/jmxtrans/model/output/OpenTSDBWriter.java

```diff
@@ -113,6 +113,9 @@ protected void startOutput() throws IOException {
 	@Override
 	protected void sendOutput(String metricLine) throws IOException {
 		try {
+			if(isDebugEnabled() || log.isDebugEnabled()){
```

Comment:
@lookfirst ,

Thanks for feedback. I have fixed the formatting issue. 

Thanks,
Bhavesh